services/wcr/migrations.md

```diff
@@ -18,7 +18,7 @@ In this case the migration of phase 1 registrations involved nothing more than a
 
 ## Multiple addresses
 
-Release [to be confirmed](https://github.com/EnvironmentAgency/waste-exemplar-frontend) included changes to the registration model so that address details are now held in a model called [address](https://github.com/EnvironmentAgency/waste-exemplar-frontend/blob/master/app/models/address.rb) which sits in an Ohm [Set](http://www.rubydoc.info/github/soveran/ohm/Ohm/Set) on the registration.
+Release [to be confirmed](https://github.com/DEFRA/waste-exemplar-frontend) included changes to the registration model so that address details are now held in a model called [address](https://github.com/DEFRA/waste-exemplar-frontend/blob/master/app/models/address.rb) which sits in an Ohm [Set](http://www.rubydoc.info/github/soveran/ohm/Ohm/Set) on the registration.
```

Comment:
```suggestion
Release [to be confirmed](https://github.com/DEFRA/waste-exemplar-frontend) included changes to the registration model so that address details are now held in a model called [address](https://github.com/DEFRA/waste-carriers-frontend/blob/master/app/models/address.rb) which sits in an Ohm [Set](http://www.rubydoc.info/github/soveran/ohm/Ohm/Set) on the registration.
```